osx/see-all-files.md

```diff
@@ -0,0 +1,37 @@
+# See all files in finder
+
+The MacOS Finder application by default is not showing you all files.
+This is OK if you are not using your computer for programin. Why to see the hidden files, sure there is a reason they are hidden.
```

Comment:
the 2nd sentence doesn't make sense

Reply:
@emmaxdan I rewrote it, please check again